galette/lib/Galette/Core/PrintLogo.php

```diff
@@ -54,4 +54,24 @@ protected function getDefaultPicture(): void
         //anyways, we have no custom print logo
         $this->custom = false;
     }
+
+    /**
+     * Returns current file full path
+     *
+     * @return string full file path
+     */
+    public function getPath(): string
+    {
+        if ($this->getFormat() !== 'webp') {
+            return $this->file_path;
+        }
+
+        //TCPDF does not support background transparency for WEBP images, create a PNG version
+        $converted_logo_path = sprintf('%s/%s.%s', GALETTE_CACHE_DIR, 'galette_printlogo_converted', 'png');
+        if (!file_exists($converted_logo_path)) {
+            $print_logo = imagecreatefromwebp($this->file_path);
+            imagepng($print_logo, $converted_logo_path);
```

Comment:
This works as expected, but a getter should not copy files... A better solution would be to automatically convert WEBP PrintLogo to PNG at upload - since WEBP version will never be used anyway.